source/codegen/metadata/nidmm/attributes.py

```diff
@@ -474,6 +474,7 @@
         'attribute_class': 'AttributeViReal64TimeDeltaSeconds',
         'channel_based': False,
         'name': 'TRIGGER_DELAY',
+        'enum': 'TriggerDelays',
```

Comment:
Document changes from all 3 files here to CHANGES.md

Reply:
updated Changes.md